nltk/parse/stanford.py

```diff
@@ -228,6 +228,7 @@ def _execute(self, cmd, input_, verbose=False):
 
         return stdout
 
+
 class StanfordParser(GenericStanfordParser):
```

Comment:
Actually, why not to rewrite the implementation of `StanfordParser` so that it uses `CoreNLP parser`, but then one needs to be careful with the CoreNLPServer.